lib/tasks/scripts/telemedicine_reports.rb

```diff
@@ -120,7 +126,10 @@ def assemble_report_data
           district[:telemed_data][:high_bs],
           district[:telemed_data][:high_bp_or_bs],
           telemed_clicks,
-          percentage(telemed_clicks, district[:telemed_data][:high_bp_or_bs])
+          district[:telemed_data][:teleconsultation_requests],
+          district[:telemed_data][:teleconsultation_records],
+          district[:telemed_data][:teleconsultation_marked_completed],
+          percentage(telemed_clicks + district[:telemed_data][:teleconsultation_requests], district[:telemed_data][:high_bp_or_bs])
```

Comment:
This is awesome. We can safely assume that total teleconsults = green button clicks plus DB records? Does the MVP's green button clicks _not_ feed into the `telemed_clicks` number?

Reply:
I'm going to check with Vinay to confirm, don't know this for sure.

Reply:
The `telemed_clicks` includes MVP clicks as well. Will change this accordingly.